src/app/items/item-by-id.component.html

```diff
@@ -1,27 +1,30 @@
-<ng-container *ngIf="state$ | async as state">
-  <ng-container *ngIf="state.isReady && state.data as itemData">
+@if (state$ | async; as state) {
+  @if (state.isReady && state.data; as itemData) {
     <ng-container *ngrxLet="isObserving$ as isObserving">
       <ng-container *ngrxLet="fullFrameContentDisplayed$ as fullFrameContentDisplayed">
-        <alg-item-header [itemData]="itemData" *ngIf="!(fullFrameContent$ | async)"></alg-item-header>
-
-        <alg-access-code-view
-          *ngIf="showAccessCodeField$ | async"
-          i18n-sectionLabel sectionLabel="Access to activity"
-          i18n-buttonLabel="Button label for joining an item by (group) code" buttonLabel="Access"
-          [itemData]="itemData"
-          (groupJoined)="reloadItem()"
-        ></alg-access-code-view>
-
-        <alg-item-permissions
-          *ngIf="observedGroup$ | async as observedGroup"
-          class="item-permissions"
-          [ngClass]="{ 'with-margin': fullFrameContentDisplayed, 'with-right-margin': !!(withLeftPaddingContentDisplayed$ | async) }"
-          [itemData]="itemData"
-          [observedGroup]="observedGroup"
-          (changed)="reloadItem()"
-        ></alg-item-permissions>
-
-        <alg-tab-bar class="tab-bar" *ngIf="!!(shouldDisplayTabBar$ | async) && !fullFrameContentDisplayed" />
+        @if (!(fullFrameContent$ | async)) {
```

Comment:
Btw, recently I've found info about condition like that. The problem is that the first of value `async` is `null` and so `false` it could be reason of render component where it shouldn't.

Here is eslint rule with description https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin-template/docs/rules/no-negated-async.md

Example: `(fullFrameContent$ | async) === false`